webui/react/src/services/decoder.ts

```diff
@@ -531,8 +531,9 @@ const messageRegex = new RegExp(
     '^',
     '\\[([^\\]]+)\\]\\s',                             // timestamp
```

Comment:
question: do we need double escaping because we're going from string to Regex?

Reply:
exactly, not the sexiest look i know :)